scripts/qa_crowbarsetup.sh

```diff
@@ -227,6 +228,7 @@ setcloudnetvars()
         ;;
         qa3)
             nodenumbertotal=8
+            nodenumber=8
```

Comment:
I thought, the intended distinction between nodenumber and nodenumbertotal was to be able to just just 2 or 3 nodes for a test while still knowing that it has more...